### ISSUE TITLE
fail to run

### ISSUE THREAD

**Issue by taozhiyuai** [z-lab/paroquant#33]
<img width="1534" height="1019" alt="Image" src="https://github.com/user-attachments/assets/19ca3f78-54a4-4cee-a6af-e50cdfa6b933" />

What I did:

pip install "paroquant[mlx]"

python -m paroquant.cli.chat --model ./Qwen3.5-4B-PARO

macOS  = 15.7.5

**Comment by jeethu** [comment]
I've got the same issue. It looks like the output logits are all `NaN`. `argmax()` returns token 0, which is "!" in the [Qwen tokenizer](https://huggingface.co/z-lab/Qwen3.5-4B-PARO/blob/main/tokenizer.json).

**Comment by liang2kl** [comment]
The error is introduced after v0.1.10. We're implementing a fix right now, and feel free to downgrade to v0.1.10 as a temporary fix:

```
pip install "paroquant[mlx]==0.1.10"
```

**Comment by Ziqiao-git** [comment]
Just a note for anyone downgrading to v0.1.10 as a temporary workaround: you may notice some performance regression. This seems to be related to the MLX bug discussed here: [ #3347](https://github.com/ml-explore/mlx/issues/3347).

**Comment by jeethu** [comment]
Fix in PR: #38.

**Comment by liang2kl** [comment]
The fix is in place, thanks @jeethu! 

Please upgrade to the latest version:

```
pip install --upgrade "paroquant[mlx]"
```
